package.json

```diff
@@ -68,6 +68,7 @@
     "compare-versions": "^3.6.0",
     "date-fns": "^2.16.1",
     "expo-barcode-scanner": "^9.1.0",
+    "expo-camera": "^11.0.3",
```

Comment:
We need the camera module now too?

Reply:
Yeah, `expo-barcode-scanner` can't do square frames on Android. They've built scanning into the main camera package now. Seems like they'll probably deprecate the other at some point.